osc-server/src/main/java/org/osc/core/broker/util/log/LogUtil.java

```diff
@@ -16,56 +16,35 @@
  *******************************************************************************/
 package org.osc.core.broker.util.log;
 
-import java.io.PrintStream;
+import java.util.Hashtable;
 
-import org.apache.log4j.Logger;
-import org.apache.log4j.PropertyConfigurator;
+import org.osgi.framework.BundleContext;
+import org.slf4j.ILoggerFactory;
 
-public class LogUtil {
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.joran.JoranConfigurator;
+import ch.qos.logback.core.joran.spi.JoranException;
 
-    public static void initLog4j() {
-        try {
-            PropertyConfigurator.configureAndWatch("./log4j.properties");
-            StdOutErrLog.tieSystemOutAndErrToLog();
-        } catch (Exception ex) {
-            System.out.println("failed to initialize log4j");
-            ex.printStackTrace();
-        }
-    }
-
-    public static class StdOutErrLog {
+public class LogUtil {
 
-        private static final Logger logger = Logger.getLogger(StdOutErrLog.class);
+    static ILoggerFactory loggerFactory;
 
-        public static void tieSystemOutAndErrToLog() {
-            System.setOut(createLoggingProxy(System.out, false));
-            System.setErr(createLoggingProxy(System.err, true));
+    public static void initLogging(BundleContext context) throws JoranException {
```

Comment:
how is the previous behaviour for tieSystemOutAndErrToLog preserved with this change?